docs/getting-started/clients/browser/index.md

```diff
@@ -8,6 +8,79 @@ sidebar_position: 3
 
 Before you start, you must complete the [Clients repository setup instructions](../index.md).
 
+## Environment Setup
+
+By default, the browser extension will run pointing to the production server endpoints. To override
+this for local development and testing, there are several options.
+
+### Using `managedEnvironment`
+
+The browser extension has the concept of a "managed environment", which is JSON configuration stored
+in
+[development.json](https://github.com/bitwarden/clients/blob/master/apps/browser/config/development.json).
```

Comment:
Dev json should probably be wrapped in `` similarly to the BrowserEnvironmentService for consistency.